src/Test/PdbUtilities/Shared/DateTimeUtilities.cs

```diff
@@ -6,8 +6,6 @@ namespace Roslyn.Utilities
 {
     internal static class DateTimeUtilities
```

Comment:
It looks like the only remaining use of this type is `ExpressionEvaluator.PdbHelpers`. We should consider removing this type and moving the one remaining method to `PdbHelpers` since it's not clear why a `Source` file depends on a `Test` helper. That can be done separately.

Reply:
Thank you! Agree. Created https://github.com/dotnet/roslyn/issues/26179 to cover this.